### DISCUSSION TITLE
OpenTelemetry resource name

### DISCUSSION THREAD

**Discussion by Wuerike** [redpanda-data/connect#2030] [Q&A] [answered]
When trying to use the otel collector to send traces to newrelic and grafana, I've noticed the resource it creates is always called benthos.

I think this happens at the line 14 from [otel.go](https://github.com/benthosdev/benthos/blob/main/internal/tracing/otel.go#L14), how complicated is change the code so this name could be configured from the config.yaml?

I'll be running several instances of benthos for different purposes and would like to be able to isolate the traces of each one.

**Comment by mihaitodor** [comment, marked as answer]
Hey @Wuerike, could you please check if this is covered by #1823? Basically, does it help if you set `service.name: foobar` in the `tags` section of the [otel config](https://www.benthos.dev/docs/components/tracers/open_telemetry_collector)?

**Comment by Wuerike** [comment]
Thanks @mihaitodor it worked! My bad I didn't checked enough throug the repo
